### PR TITLE
task NXP 32953 k8s app package

### PR DESCRIPTION
**Pull request by ataillefer**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1 @@
+FROM gcr.io/cloud-marketplace-tools/k8s/deployer_helm/onbuild
```
Comment (kevinleturc) [inline Dockerfile]: nitpick: new line at the end?

File: chart/nuxeo-mp/logo.png
Comment (kevinleturc) [inline chart/nuxeo-mp/logo.png:1]: Should we add .gitattributes for images (good practice)?

Now that LFS is part of git, it should just be:
```
git lfs track "*.png"
```

File: chart/nuxeo-mp/requirements.yaml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+dependencies:
+- name: nuxeo
+  version: 3.1.7-PR-68-BUILD-6
+  # version: ~3.1.6
+  repository: https://chartmuseum.platform.dev.nuxeo.com/
+  # repository: https://packages.nuxeo.com/repository/helm-releases-public/
```
Comment (kevinleturc) [inline chart/nuxeo-mp/requirements.yaml]: Comment just to not forget it.

**Review by kevinleturc — APPROVED** [review]
Good work!

**Comment by ataillefer** [comment]
Added tester pod and completed application descriptor.

**Review by kevinleturc — APPROVED** [review]

**Pull request merged**